lib/projects/people-and-code/templates/src/index.html

```diff
@@ -10,9 +10,7 @@
   <body class="body">
     <header>
       <p>
-        <a href="#main" tabindex="0" class="local"
-          >Jump link to the main content</a
-        >
+        <a href="#main" class="skip">Skip to main content</a>
       </p>
       <h1>Welcome</h1>
     </header>
```

Comment:
Should we add the byline in the main section? Both here and on the Sitemap?

Reply:
We've got a snippet for it in the website repo

Reply:
Good catch.

I think it makes sense to add the snippets to a new P&C project (which I've now done) but not to include the byline in the example pages because this is a generic P&C project template. But if you think they should be included, I'm easy.

I've also updated the descriptions to mention parcel where appropriate.

Reply:
Agreed, the byline should only be on the front page of examples.